.gitignore

```diff
@@ -39,9 +39,9 @@ vendor
 # Ignore temporary *.swp files
 *.swp
 
-<<<<<<< HEAD
 # Ignore generated self-signed TLS files from makefile.
 tls/**
-=======
+
+# Ignore IDE files
```

Comment:
Np in adding them, but there's also another way I'd recommend to use locally for each of us: https://stackoverflow.com/questions/7335420/global-git-ignore :)